server/src/web/user/user.controller.ts

```diff
@@ -0,0 +1,46 @@
+import {
+    Body,
+    Controller,
+    Delete,
+    Get,
+    Inject,
+    Injectable,
+    Param,
+    Post, Put,
+    Query,
+} from '@nestjs/common';
+import {User} from '../../domain/user/user.entity';
+import {UserDto} from '../../domain/user/dto/user.dto';
+import {DeleteResult} from 'typeorm';
+import {UserService} from '../../app/user/user.service';
+
+@Injectable()
+@Controller('users')
+export class UserController {
+    constructor(@Inject('UserService') private service: UserService) {}
+
+    @Get(':id')
+    async get(@Param('id') id: number): Promise<User> {
+        return await this.service.get(id);
+    }
+    @Post()
+    async create(@Body() userDto: UserDto): Promise<User> {
+        return await this.service.create(userDto);
+    }
+    @Delete(':id')
+    async delete(@Param('id') id: number): Promise<DeleteResult> {
+        return await this.service.delete(id);
+    }
+    @Put(':id/increase-level')
```

Comment:
Need Comment: @junbeomlee 
not sure http method and url is good.

i referenced: [github-api](https://developer.github.com/v3/pulls/#update-a-pull-request-branch)
`PUT /repos/:owner/:repo/pulls/:pull_number/update-branch`

it was using verb in url